.vscode/tasks.json

```diff
@@ -4,16 +4,81 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": "build",
+            "label": "build-tsc",
```

Comment:
Heh, guess you can tell how often I actually use these... I always just have `yarn build:tsc` running in a terminal.